notebooks/SpecvizExample.ipynb

```diff
@@ -190,17 +173,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# axis 1 corresponds to the Y-axis and axis 0 to the X-axis\n",
     "# fmt can be '0.1e' to set scientific notation or '0.2f' to turn it off\n",
-    "specviz.set_spectrum_tick_format(fmt=\"0.2f\", axis=1)"
+    "specviz.viewers['spectrum-viewer'].set_tick_format(fmt=\"0.2f\", axis='x')"
```

Comment:
While this is reasonable in the context of "de-configging," user might see this as a step backwards because there is more characters to type out. I wonder if the change log should elaborate more on the motivation.

Reply:
any specific suggestions for wording?

Reply:
"to facilitate config-agnostic framework planned for the future" or something like that?

Reply:
added "as configs are centralized" - does that give enough context without being too technical?